src/50_Samples_%26_Templates/Live_Blog.html

```diff
@@ -173,11 +187,19 @@
     </div>
     <!-- #### Blog Posts -->
     <!-- Use amp-live-list to implement a live blog. Find an amp-live-list example [here](/components/amp-live-list) -->
```

Comment:
Can you explain the pagination feature? Or do you plan to add this to the amp-live-list sample?